python/src/otel/otel_sdk/opentelemetry/resource/__init__.py

```diff
@@ -0,0 +1,19 @@
+from os import environ
+from opentelemetry.sdk.resources import (
+    Resource,
+    ResourceDetector,
+)
+
+
+class AwsLambdaResourceDetector(ResourceDetector):
```

Comment:
Guessing we'd add resource detectors like this and more here

https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/sdk-extension/opentelemetry-sdk-extension-aws

Reply:
yes, will move to python contrib with aws instrumentor, if Lambda resource detector is accepted by Python maintainer :)